packages/compiler/core/formatter.ts

```diff
@@ -1,5 +1,6 @@
 import prettier from "prettier";
 import * as typespecPrettierPlugin from "../formatter/index.js";
+export { printId as formatIdentifier } from "../formatter/print/printer.js";
```

Comment:
Is this sufficient? When I try and import, I don't get `"@typespec/compiler/formatter"` as an option. I only get `emitter-framework`, `module-resolver` and `testing` as options.

Reply:
It seems like either `"./formatter"` needs to be added to the exports in package.json or that this method needs to be imported in `core/index.ts`.

Reply:
Actually, it looks like the formatter is exported from index.js.